src/System.Net.Security/src/System/Net/Security/SslStreamPal.OSX.cs

```diff
@@ -74,6 +102,12 @@ public static void VerifyPackageInfo()
             return new SafeFreeSslCredentials(certificate, protocols, policy);
         }
 
+        internal static byte[] GetNegotiatedApplicationProtocol(SafeDeleteContext context)
+        {
+            // OSX SecureTransport does not export APIs to support ALPN, no-op.
```

Comment:
It's available on newer versions, correct? Will there be a way to light that up?

Reply:
It should be possible to just P/Invoke into the function; though appropriate guards would need to be put in place for detecting suitability.  I'd prefer it was shimmed, but sometimes needs must.